pkg/ipmi/ipmi.go

```diff
@@ -77,6 +82,34 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type StandardSelEventRecord struct {
+	Timestamp    uint32
+	GenId        uint16
+	EvmRev       uint8
+	SensorType   uint8
+	SensorNum    uint8
+	EventTypeDir uint8
+	EventData    [3]uint8
+}
+
+type OemTsSelEventRecord struct {
+	Timestamp        uint32
+	ManfId           [3]uint8
+	OemTsDifinedData [6]uint8
+}
+
+type OemNotsSelEventRecord struct {
+	OemNotsDifinedData [13]uint8
+}
+
+type SelEventRecord struct {
```

Comment:
Can this just be called `Event`? Is there an event system other than SEL? (if so, remove Sel)